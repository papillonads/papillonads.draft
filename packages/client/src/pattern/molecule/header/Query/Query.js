import React from 'react'

const Query = () => <div />

export default Query
