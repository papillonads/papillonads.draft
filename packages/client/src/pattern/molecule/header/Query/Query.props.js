import { bool } from 'prop-types'

export const queryDefaultProps = {
  className: '',
}

export const queryPropTypes = {
  className: bool,
}
