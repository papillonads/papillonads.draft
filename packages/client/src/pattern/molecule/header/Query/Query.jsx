import React from 'react'
import cn from 'classnames'
import { queryDefaultProps, queryPropTypes } from './Query.props'

const Query = ({ className }) => {
  const queryClassList = cn(className)

  return <div className={queryClassList}>Query</div>
}

Query.defaultProps = queryDefaultProps

Query.propTypes = queryPropTypes

export default Query
