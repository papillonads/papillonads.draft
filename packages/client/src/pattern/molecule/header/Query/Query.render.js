import React from 'react'
import Query from './Query'

const defaultRender = <Query />

export { defaultRender }
