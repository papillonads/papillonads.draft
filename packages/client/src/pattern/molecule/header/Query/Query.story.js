import { storiesOf } from '@storybook/react'
import { defaultRender } from './Query.render'
import { getInfo } from '../../../../../../../.storybook/utility'

const defaultRenderInfo = getInfo('default')

storiesOf('Patttern/Molecule/Header/Query', module).add(defaultRenderInfo.text, () => defaultRender, defaultRenderInfo.parameters)
