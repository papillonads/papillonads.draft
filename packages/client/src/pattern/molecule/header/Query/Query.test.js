import { defaultRender } from './Query.render'

describe('<Query />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
