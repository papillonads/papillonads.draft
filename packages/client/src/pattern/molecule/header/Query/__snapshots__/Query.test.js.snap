// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Query /> Snaphot must match defaultRender 1`] = `
<div
  className="query"
>
  Query
</div>
`;
